library/ssl_tls.c

```diff
@@ -7131,6 +7154,12 @@ static int ssl_tls12_populate_transform( mbedtls_ssl_transform *transform,
         ssl_mode == MBEDTLS_SSL_MODE_CBC ||
         ssl_mode == MBEDTLS_SSL_MODE_CBC_ETM )
     {
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+        size_t block_size = PSA_BLOCK_CIPHER_BLOCK_MAX_SIZE;
```

Comment:
I think that happens to be correct in practice now because all block ciphers usable in TLS have the same block size, but as a matter of principle I think something like `PSA_BLOCK_CIPHER_BLOCK_LENGTH( key_type );` would be more correct.